images/win/scripts/Installers/Install-MongoDB.ps1

```diff
@@ -6,7 +6,13 @@
 # Install mongodb package
 $toolsetVersion = (Get-ToolsetContent).mongodb.version
 $latestChocoPackage = Get-LatestChocoPackageVersion -TargetVersion $toolsetVersion -PackageName "mongodb"
-Choco-Install -PackageName mongodb -ArgumentList "--version=$latestChocoPackage"
+
+$installDir = "c:\PROGRA~1\MongoDB"
+$downloadURL = "https://fastdl.mongodb.org/windows/mongodb-windows-x86_64-$latestChocoPackage-signed.msi"
+$downloadPath = "C:\temp\mongodb.msi"
+Invoke-Webrequest -URI $downloadURL -OutFile ( New-Item -Path $downloadPath -Force )
+cmd /c start /wait msiexec /q /i $downloadPath INSTALLLOCATION=$installDir ADDLOCAL="all"
```

Comment:
we should use the `Install-Binary` helper

Reply:
fixed